mmdet/models/detectors/single_stage_seg.py

```diff
@@ -0,0 +1,110 @@
+import torch.nn as nn
+
+from mmdet.core import bbox2result
+from ..builder import DETECTORS, build_backbone, build_head, build_neck
+from .base import BaseDetector
+
+
+@DETECTORS.register_module()
+class SingleStageSegDetector(BaseDetector):
```

Comment:
Is it possible that we do not write a new `SingleStageSegDetector` but modify the existing `SingleStageDetector` to have a `mask_head`? It could make the `SingleStageDetector` more general and it seems we only need to add a few lines of code for `mask_head`.